Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -79,6 +79,96 @@ class Foo < Formula
       end
     end
 
+    describe "#audit_license" do
+      let(:spdx_data) {
+        full_path = File.join(File.dirname(__FILE__), "../../data/spdx.json")
+        File.open(full_path, "r") do |f|
+          JSON.parse(f.read)
+        end
```

Comment:
```suggestion
        Pathname(File.join(File.dirname(__FILE__), "../../data/spdx.json")).read
```

or similar using `Pathname`